packages/fx-core/tests/plugins/resource/apiconnector/expectedFiles/js/aad.js

```diff
@@ -0,0 +1,30 @@
+const teamsfxSdk = require("@microsoft/teamsfx");
+
+// Loads current app's configuration
+const teamsFx = new teamsfxSdk.TeamsFx(teamsfxSdk.IdentityType.App);
+// Initializes a new axios instance to call fake API
+const appCredential = teamsFx.getCredential();
+const authProvider = new teamsfxSdk.BearerTokenAuthProvider(
+  // please replace ‘<your-api-scope>’ with actual api scope value.
+  // You can visit https://aka.ms/teamsfx-connect-api to understand how to call your API with TeamsFx SDK.
```

Comment:
Let's remove this comment, since we have mentioned how to make requests as well as this document below. And we expect users focus on the udpate <your-api-scope> comment here.